services/federator/test/integration/Test/Federator/InwardSpec.hs

```diff
@@ -126,7 +128,7 @@ spec env =
     -- and "IngressSpec".
     it "rejectRequestsWithoutClientCertInward" $
       runTestFederator env $ do
-        originDomain <- originDomain <$> view teTstOpts
+        originDomain <- (.originDomain) <$> view teTstOpts
```

Comment:
was that necessary for some reason?  (i'm not opposed to your version, just to unmotivated changes. :)

Reply:
Yes, it helps GHC figure out that we want to use the new record field methods. Otherwise it gets very confused about what the type should be.